frontend/src/components/session/SessionHistory.tsx

```
@@ -0,0 +1,369 @@
+    }
+
+    // Apply sorting
+    filtered.sort((a, b) => {
+      let comparison = 0;
+
+      switch (sortBy) {
+        case 'date':
+          comparison = a.lastActivity.getTime() - b.lastActivity.getTime();
+          break;
+        case 'name':
+          const nameA = a.session.session_name || '';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)